### PR TITLE
Update overview to include information about alerts during interruptions.mdx

### PR DESCRIPTION
**Pull request by esutherland4**
Add information related to alert notifications and service interruptions - It appears a previous pull request entry has been removed or the page was deleted? https://github.com/newrelic/docs-website/pull/9707

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add information related to alert notifications and service interruptions - It appears a previous pull request entry has been removed or the page was deleted? https://github.com/newrelic/docs-website/pull/9707
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/alerts/overview.mdx [hunk 1]
```diff
@@ -187,7 +187,7 @@ This diagram shows how alerts work: you create an alert policy that includes an
         </tr>        
     </tbody>
 </table>
-
+Note: During a service interruption, alerts may be unavailable.
```
Comment (ally-sassman) [inline src/content/docs/alerts/overview.mdx:190]: @esutherland4 Can we say more about this, or potentially change the phrasing? I would think that a service interruption is exactly when I'd want to get alerted about it

Comment (esutherland4) [reply src/content/docs/alerts/overview.mdx:190]: @ally-sassman Hey! It's referencing New Relic service interruptions, so maybe below is better? This is legal requested language (related to when we suppress alerts during incidents) that we had in the docs before that was removed for some reason. Here is the old PR: https://github.com/newrelic/docs-website/pull/9707

> During a New Relic service interruption, alerts may be temporarily unavailable.

Comment (ally-sassman) [inline src/content/docs/alerts/overview.mdx:190]:
```suggestion
Note: During a New Relic service interruption, alerts may be temporarily unavailable.
```

**Review by ghost — DISMISSED** [review]

**Comment by ally-sassman** [comment]
@esutherland4 Can you please sign the CLA above so I can merge? Thanks!

**Review by ghost — APPROVED** [review]

**Comment by ally-sassman** [comment]
Created a new PR here: https://github.com/newrelic/docs-website/pull/18879

**Review by ghost — APPROVED** [review]

**Pull request merged**